<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Najmlodszy">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Najmłodszy żołnierz</string>
    </property>
   </widget>
   <widget class="QLabel" name="WyswietlNajmlodszego">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_najmlodszy">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetl najmłodszego</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dodajZolnierza">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj Zołnierza</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Szeregowy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pułkownik</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Generał</string>
     </property>
    </item>
   </widget>
   <widget class="QListWidget" name="ListaZolnierzy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
